packages/beeq/src/components/date-picker/bq-date-picker.tsx

```diff
@@ -305,11 +328,20 @@ export class BqDatePicker {
     this.open = this.type === 'multi';
   };
 
+  private handleCalendarRangeStart = () => {
+    this.hasRangeEnd = false;
+  };
```

Comment:
If we add a new `tentative` property, we should guarantee that `tentative` gets the range start value

```suggestion
  private handleCalendarRangeStart = (ev: CustomEvent) => {
    this.hasRangeEnd = false;
    this.tentative = new Date(ev.detail).toLocaleDateString('fr-CA');
  };
```